src/main/java/main/TaskList.java

```diff
@@ -0,0 +1,74 @@
+package main;
+
+import java.util.ArrayList;
+
+import task.Task;
+
+/**
+ * Stores the task into a TaskList
+ */
+public class TaskList {
+    private ArrayList<Task> list;
```

Comment:
May want to rename this to distinguish clearly as a multi-valued variable. 
e.g. `tasks`